tests/test_snapshot.py

```diff
@@ -61,7 +76,6 @@ def test_track_writes(self):
         os.sync()
         info = elastio_snap.info(self.minor)
         start_nr = info["nr_changed_blocks"]
-        self.assertNotEqual(start_nr, 0)
```

Comment:
What about to leave this check for fs not equal to `xfs`?

Reply:
Potentially this can be `0` on other filesystems as well.